packages/geoprocessing/src/toolbox/overlapFeatures.ts

```diff
@@ -56,11 +56,33 @@ export async function overlapFeatures(
   const { includeChildMetrics } = newOptions;
   let sumValue: number = 0;
   let isOverlap = false;
+  let featureIndices: Set<number> = new Set();
   const sketches = (Array.isArray(sketch) ? sketch : toSketchArray(sketch)).map(
     (s) => truncate(s)
   );
   const finalFeatures = features.map((f) => truncate(f));
 
+  // Create individual sketch metrics
+  const sketchMetrics: Metric[] = sketches.map((curSketch) => {
+    const sketchValue = doIntersect(
```

Comment:
The return is now more complex, and the variable name `sketchValue` seems incorrect?  What about `intersections`?

Reply:
I like `intersections` as the new variable name.